pkgs/swift2objc/lib/src/config.dart

```diff
@@ -32,11 +34,19 @@ class Config {
   /// intermediate files after generating the wrapper
   final Uri? tempDir;
 
+  /// Filter function to filter APIs 
+  /// 
+  /// APIs can be filtered by name
+  /// 
+  /// TODO: Add `excludeAll` option to exclude all or include all declarations
+  final bool Function(Declaration declaration)? filter;
```

Comment:
The name and documentation don't really make it clear whether this function should return true or false to include the given declaration. Maybe rename it `include`. Also, make it non-null and then set a default in the constructor.